odoo/addons/base/models/res_lang.py

```diff
@@ -332,6 +332,21 @@ def format(self, percent, value, grouping=False, monetary=False):
 
         return formatted
 
+    def copy(self, default=None):
+        self.ensure_one()
+        default = dict(default or {})
+
+        duplicates = self.search([("name", "=like", f"{self.name} (copy)")])
+        if duplicates:
+            raise UserError(_("There is already a duplicate of this language. You can't create two duplicates from the same language."))
```

Comment:
Drop this part, it's doing the same thing as the sql_constraints, which will be more efficient anyways